### PR TITLE
MEN-4094: add Overview/Security/Remote Terminal subsection

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Issue: https://tracker.mender.io/browse/MEN-4094

### CODE REVIEW

File: 02.Overview/11.Security/docs.md [hunk 1]
```diff
@@ -108,3 +108,18 @@ any other HSM access methods that are supported by
...
+This feature is called [Remote Terminal](../../02.Overview/14.Remote.Terminal/docs.md).
+Remote Terminal is an optional feature.
+One part of the Remote terminal configuration is the username of the user on the device for whom the shell will be created.
+Using Remote Terminal you can execute any command the user on the device can execute.
+Like with any other operation, the Mender backend will allow only authenticated Mender users to access the Remote Terminal.
+You can apply additional restrictions on which Mender users can access Remote Terminal using RBAC.
+The [mender-shell](https://github.com/mendersoftware/mender-shell), part of the Remote Terminal that is running on the device,
+obtains device token through DBus API. It is the same token Mender client is using.
+The mender-shell uses [Encrypted WebSocket connections](https://tools.ietf.org/html/rfc6455#section-11.1.2) to communicate with the server.
+In general, the security impact of enabling Remote Terminal is similar to the one when enabling ssh.
```
Comment (merlin-northern) [inline 02.Overview/11.Security/docs.md:123]: with one crucial difference: there is not port open on a device.
suggested: explicitly mention that fact. maybe mention also that it is impossible for anyone to initiate a connection.

**Review by drewmoseley — APPROVED** [review]

**Comment by kjaskiewiczz** [comment]
this PR needs https://github.com/mendersoftware/mender-docs/pull/1242

**Comment by estenberg** [comment]
This is good information I think, but as mentioned in #1242 we should perhaps move it to a separate Add on chapter (sorry :-/)

**Comment by merlin-northern** [comment]
included in https://github.com/mendersoftware/mender-docs/pull/1242

**Pull request closed without merging**